ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -587,7 +587,7 @@ private DependencyObject ResolveView(int tag)
             return view;
         }
 
-        private IViewManager ResolveViewManager(int tag)
+        public IViewManager ResolveViewManager(int tag)
```

Comment:
maybe. I'd need to look at the consumption of the methods being exposed, but it's not jumping out at me in this diff. Also, since its only two methods and they take a POD and return a pure interface, there isn't a coupling problem -- just an ISP problem.